docs/docs/modules/memory/index.mdx

```diff
@@ -0,0 +1,9 @@
+---
+sidebar_label: Memory
+---
+
+import DocCardList from "@theme/DocCardList";
+
+# Getting Started: Memory
```

Comment:
Let's add some more context in this page

Reply:
Done